sdk/confidentialledger/azure-resourcemanager-confidentialledger/pom.xml

```diff
@@ -51,5 +51,30 @@
       <artifactId>azure-core-management</artifactId>
       <version>1.8.0</version> <!-- {x-version-update;com.azure:azure-core-management;dependency} -->
     </dependency>
+    <!-- Test dependencies-->
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
+      <version>1.6.0</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
```

Comment:
usually you only need "junit-jupiter-engine", e.g. https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/hybridnetwork/azure-resourcemanager-hybridnetwork/pom.xml#L68